games/devtest/mods/unittests/bulk.lua

```diff
@@ -0,0 +1,16 @@
+
+-- Test bulk get and set
+local function test_bulk_get_set(_,pos)
+	local positions = {}
+	for n = 0, 10, 1 do
+		table.insert(positions, vector.add(pos, vector.new(0, n, 0)))
+	end
+	local nodes = minetest.bulk_get_node(positions)
+
+	minetest.bulk_set_node(positions, {name="basenodes:stone"})
+
+	for index, set_pos in pairs(positions) do
+		minetest.set_node(set_pos, nodes[index])
+	end
+end
```

Comment:
this test is pointless, it doesn't have any asserts

Reply:
fixed